### DISCUSSION TITLE
error editable_checkbox: With editable-columns got javascript error: registerMinorEditInputEvents is not defined

### DISCUSSION THREAD

**Discussion by kevinartimedia** [Laravel-Backpack/community-forum#736] [Q&A (Help)]
Hello everyone!
We have the following problem: The **editable_checkbox** is not displayed correctly and also when we click on the button it shows this error.

thanks!

![image](https://github.com/Laravel-Backpack/community-forum/assets/112622409/179d67e6-afe8-424d-a7d6-54c3de7a375b)

**Comment by tabacitu** [comment]
Hi @kevinartimedia ,

Hmm weird. Should be some implementation problem, because I see it's working on my computer and in [our online demo](https://demo.backpackforlaravel.com/admin/editable-monster):
![CleanShot 2023-11-08 at 13 03 25](https://github.com/Laravel-Backpack/community-forum/assets/1032474/08630d22-ec49-4473-a6cd-30a507569767)


I don't have much info to go with here, so... here's are some shots in the dark: 

**1. Have you seen https://github.com/Laravel-Backpack/CRUD/issues/4818 ? They had overridden the blade layout, but hadn't copied everything. Do you use a custom layout - could this be the problem for you?**

**2. Have you made any changes to the JS that gets loaded? I [see here that could also cause this error](https://stackoverflow.com/questions/76794907/uncaught-referenceerror-registerminoreditinputevents-is-not-define-facing-js-is).** 

If none of the above fix it for you, please provide more info:
- the output of `php artisan backpack:version`
- what customizations you have made to your admin panel (custom layouts, custom blade files etc)

Cheers!

**Reply by tabacitu** [reply]
Also an interesting test - the `editable_checkbox` doesn't work... but if you change one of your text columns to `editable_text`... do those work? Or same problem, don't load the JS?

**Comment by jcastroa87** [comment]
We haven’t heard back from you in a while, so we’re closing this issue for now. If you still need assistance, please don’t hesitate to reopen the issue or create a new one.

Thank you, and take care!

Cheers.
